### PR TITLE
Update toolbar.js #25934

### PR DESCRIPTION
**Pull request by Nagamaiah007**
### Fixed Issues (if relevant)

1. magento/magento2#25934: chrome initiate two times toolbar.js

### Manual testing scenarios (*)

1. got to product list, search or advance search page
2. open developer tools -> network(tab)(Chrome browser)
3. Select/click any of the toolbar option like Grid/List, Sort By, or show X per page
4. Network(Chrome Browser) selected/clicked link triggering two time and because of two time trigger backed doing twice process after canceled one also.
   (for verify put any log in to catalog->product->list controller).
5. By this fix issue is fixed

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: app/code/Magento/Catalog/view/frontend/web/js/product/list/toolbar.js [hunk 1]
```diff
@@ -34,10 +34,14 @@ define([
...
         /** @inheritdoc */
         _create: function () {
+            if (window.isToolbarLoaded) {
+                return;
+            }
             this._bind($(this.options.modeControl), this.options.mode, this.options.modeDefault);
             this._bind($(this.options.directionControl), this.options.direction, this.options.directionDefault);
             this._bind($(this.options.orderControl), this.options.order, this.options.orderDefault);
             this._bind($(this.options.limitControl), this.options.limit, this.options.limitDefault);
+            window.isToolbarLoaded = true;
```
Comment (krzksz) [inline app/code/Magento/Catalog/view/frontend/web/js/product/list/toolbar.js]: Storing this in a global variable is not needed, you can move it to a local variable above widget declaration.

**Review by krzksz — CHANGES_REQUESTED** [review]

File: app/code/Magento/Catalog/view/frontend/web/js/product/list/toolbar.js (hunk 1 above)
Comment (Nagamaiah007) [reply app/code/Magento/Catalog/view/frontend/web/js/product/list/toolbar.js]: I tried this (this.options.isToolbarLoaded) earlier it is not working as expected so, had to use it as a global variable.

Comment (krzksz) [reply app/code/Magento/Catalog/view/frontend/web/js/product/list/toolbar.js]: I mean adding `var isToolbarLoaded` at line 11 instead: https://github.com/magento/magento2/pull/26097/files#diff-1364732a1dafeccded6675a06411dbdbR11

I also think that changing the wording to `isTollbarInitialized` would be more informative.

**Review by krzksz — CHANGES_REQUESTED** [review]

**Comment by ptylek** [comment]
@magento run all tests

**Review by ptylek — DISMISSED** [review]

**Comment by magento-engcom-team** [comment]
Hi @ptylek, thank you for the review.
[ENGCOM-7259](https://jira.corp.magento.com/browse/ENGCOM-7259) has been created to process this Pull Request
:eight_spoked_asterisk: @ptylek, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Review by engcom-Alfa — CHANGES_REQUESTED** [review]
Hi @Nagamaiah007  .
     Due to [Magento Definition of Done](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing_dod.html) all code must be covered by tests. Please cover your fix by jasmine test.
Thanks!

**Comment by krzksz** [comment]
@magento run Database Compare

**Comment by krzksz** [comment]
@magento run all tests

**Comment by VladimirZaets** [comment]
@Nagamaiah007, I am closing this PR now due to inactivity.
Please reopen and update if you wish to continue.
Thank you for collaboration

**Pull request closed without merging**